federation/BUILD

```diff
@@ -26,6 +26,7 @@ filegroup(
         "//federation/cmd/genfeddocs:all-srcs",
         "//federation/cmd/kubefed:all-srcs",
         "//federation/develop:all-srcs",
+        "//federation/pkg/crud:all-srcs",
```

Comment:
Or, is part of this change reconceptualizing the CRUD machinery as something to use in production code?